web-api/src/persistence/elasticsearch/getDocketNumbersByStatusAndByJudge.ts

```diff
@@ -1,7 +1,7 @@
 import {
   CavAndSubmittedCaseResponseType,
   JudgeActivityReportCavAndSubmittedCasesRequest,
-} from '../../../../web-client/src/presenter/judgeActivityReportState';
+} from '@web-client/presenter/judgeActivityReportState';
 import { MAX_ELASTICSEARCH_PAGINATION } from '@shared/business/entities/EntityConstants';
```

Comment:
I think these types should live in the shared directory, to avoid importing from web-client into web-api

Reply:
Agreed. Fixed in this commit: [c92f38b](https://github.com/ustaxcourt/ef-cms/pull/3841/commits/c92f38bb9624d72b9b2bdbb7a431abb696ca1f8a)